index.css

```diff
@@ -0,0 +1,45 @@
+body{
+  font-family: monospace;
+  background-color: lavender;
+  display: flex;
+  flex-direction: column;
+  align-items: center; 
+}
+
+#header{
+  width:70%;
+  height:150px;
+  margin-bottom: 30px;
+  padding:10px;
+  background-color: skyblue;
+  display:flex;
+  align-items: center;
+  position: relative;
+}
+
+img{
+  height:inherit;
+  float:left;
+}
+
+h1{
+  position: absolute;
```

Comment:
* Logo image overflows the header container and the header text gets over the image on a smaller screen width. Check in with @lakerfan1994 for an idea of how to approach this.
* Make sure to checkout how to use media queries for responsiveness